examples/zenohcxx/universal/z_delete.cxx

```diff
@@ -33,6 +33,7 @@ int _main(int argc, char **argv) {
             exit(-1);
         }
     }
+#endif
```

Comment:
Does the binding over Zenoh-Pico implementing the config helpers?

Reply:
When using `zenoh-c`, we are setting the `Zenoh Peer/Router` to connect to. Why not to have an `#ifdef` that does the same with the zenoh-pico config way?

For example:
```
    if (argc > 3) {
#ifdef ZENOHCXX_ZENOHC
        if (!config.insert_json(Z_CONFIG_CONNECT_KEY, argv[3])) {
            printf(
                "Couldn't insert value `%s` in configuration at `%s`. This is likely because `%s` expects a "
                "JSON-serialized list of strings\n",
                argv[3], Z_CONFIG_CONNECT_KEY, Z_CONFIG_CONNECT_KEY);
            exit(-1);
        }
#elif ZENOHCXX_ZENOHPICO
        if (!config.insert(Z_CONFIG_LISTEN_KEY, argv[2])) {
            printf(
                "Couldn't insert value `%s` in configuration at `%s`. This is likely because `%s` expects a "
                "JSON-serialized list of strings\n",
                argv[2], Z_CONFIG_LISTEN_KEY, Z_CONFIG_LISTEN_KEY);
            exit(-1);
        }
#endif
    }

Reply:
Done